codingtovictory/day6/takeHomeChallenge/style.css

```diff
@@ -0,0 +1,46 @@
+.FirstA {   
+    font-size: 25px;
+    padding-left: 20px;
+
+}
+
+.SecondA { 
+    font-size: 25px;
+    padding-left: 20px;
+    margin-top: 20px;
+
+}
+
+.ThirdA { 
+    font-size: 25px;
+    padding-left: 20px;
+    margin-top: 20px;
+
+}
+
+a { 
+    color: antiquewhite;
+
+}
+
+a:visited { 
+    color: purple;
+    text-decoration: double;
+
+}
+
+a:hover { 
+    color: blue;
+
+}
+
+.RedirectA {
+    font-size: 25px;
+    text-align: center;
+    padding-top: 20px;
+}
```

Comment:
Since this is the only element that is given this styling and id should be utilized. Id is used if just one element is getting styling applied to it and classes are used if more than one element is getting the same styling applied. In your html file just use id instead of class and to call it in your css file you would use the #, ex.:
```
#RedirectA{
    font-size: 25px;
    text-align: center;
    padding-top: 20px;
}

Reply:
Looks like I utilized it backwards then, I'll keep that in mind going forward. Thank @OnkurLal